### DISCUSSION TITLE
PR comment & approve button ordering inconsistent

### DISCUSSION THREAD

**Discussion by nkanetka** [community/community#78806] [Mobile] [answered]
I noticed the location of the PR approve and comment buttons is inconsistent between mobile and web. Slightly annoying as I sometimes go between the 2 platforms.

![image](https://github.com/community/community/assets/10577783/b40b1bf5-9f62-44c9-bc1f-9e39accafe28)

![image](https://github.com/community/community/assets/10577783/d41b5964-7a05-4a1c-8fda-4d6db7ab8a40)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.142.0 (117666737)
Device: iPhone 13 Pro (iOS 17.1.2)
</details>

**Comment by stevepopovich** [comment, marked as answer]
Hi @nkanetka very fair feedback. I will pass this on to the team for consideration. Thanks for the feedback!

**Reply by Crown25352** [reply]
✌More articles

**Reply by metaturso** [reply]
Go figure! I wasn't even aware that the mobile PR screen came with an option to submit a review. Despite my multiple attempts, I've never succeeded in submitting a code review through the app.

But, joking aside, the PR screen on mobile Android is almost impossible to use effectively. It's somewhat useful for a quick glance at the code and comments, but even those features can be challenging to access because of all the UI noise.

It doesn't help that I'm seemingly incapable of finding docs for the GitHub mobile app. The only articles on GitHub.com when I search "GitHub mobile approve PR" or "GitHub mobile submit code review" don't exactly answer my question:
- https://github.blog/2020-10-14-even-better-code-review-in-github-for-mobile/
- https://github.blog/changelog/2021-05-12-working-with-pull-requests-on-github-mobile-is-now-much-easier/
- https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews

Even Bard/Bing can't find a usable answer to this question.

Please improve the docs for GitHub Mobile apps, then consider improving the PR review screen keeping in mind that people use it primarily to skim through the code, read comments and leave new ones (batch) and occasionally submit a review.

**Reply by stevepopovich** [reply]
Hi @metaturso thanks for the feedback. I agree with a lot of what you are saying here and will pass it on to the team for consideration. Thank you again for writing in!
